main.tf

```diff
@@ -820,6 +820,39 @@ resource "aws_vpc" "agentless_scan_vpc" {
   }
 }
 
+resource "aws_default_network_acl" "default" {
+  count = var.regional && !var.use_existing_vpc ? 1 : 0
+  default_network_acl_id = aws_vpc.agentless_scan_vpc[0].default_network_acl_id
+
+  egress {
+    protocol   = -1
+    rule_no    = 100
+    action     = "allow"
+    cidr_block = "0.0.0.0/0"
+    from_port  = 0
+    to_port    = 0
+  }
+
+  ingress {
```

Comment:
this is more a measure of my ignorance than anything...
why allow ingress from port 443? 
Do we run an https server inside this VPC? 
What about port 1024?

Reply:
From our huddle:
- it's a port range 1024-65K
- needed for return traffic into our ephemeral ports